terraform/groups/ecs-service/main.tf

```diff
@@ -0,0 +1,76 @@
+provider "aws" {
+  region = var.aws_region
+}
+
+terraform {
+  backend "s3" {
+  }
+  required_version = "~> 1.3"
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "~> 4.54.0"
+    }
+    vault = {
+      source  = "hashicorp/vault"
+      version = "~> 3.18.0"
+    }
+  }
+}
+
+module "ecs-service" {
+  source = "git@github.com:companieshouse/terraform-modules//aws/ecs/ecs-service?ref=1.0.289"
```

Comment:
I think this is the latest terraform module ecs release version.
https://github.com/companieshouse/terraform-modules/releases/tag/1.0.296
Can we use 1.0.296 here?